frappe/desk/page/activity/activity.py

```diff
@@ -13,14 +13,18 @@ def get_feed(start, page_length):
 	match_conditions_comment = get_feed_match_conditions(frappe.session.user, 'Comment')
 
 	result = frappe.db.sql("""select X.*
-		from (select name, owner, modified, creation, seen, comment_type,
-				reference_doctype, reference_name, link_doctype, link_name, subject,
-				communication_type, communication_medium, content
+		from (select `tabCommunication`.name, `tabCommunication`.owner, `tabCommunication`.modified,
+				`tabCommunication`.creation, `tabCommunication`.seen, `tabCommunication`.comment_type,
```

Comment:
Use table alias

Reply:
@netchampfaris Cannot alias as data is fetched from `get_feed_match_conditions` for both Communication and Comment